frontend/src/API/InventoryAPI.js

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+
+export const getItems = () => {
+    return axios({
+        method: 'get',
+        url: '/api/supplies/'
+    });
+}
```

Comment:
return axios.get('/api/supplies);
axios.post(...) axios.delete(...)
you don't have to change it but for future use.

Reply:
I decided to do it this way for readability. Subjective matter.